packages/web-app-files/tests/unit/helpers/resources.spec.js

```diff
@@ -0,0 +1,138 @@
+        it.todo(
+          'should return "true" if "Deletable" property exists in the resource permissions array'
+        )
+        it.todo(
+          'should return "false" if "Deletable" property does not exist in the resource permissions array'
+        )
+      })
+
+      describe('canRename', () => {
+        it.todo(
+          'should return "true" if "Renameable" property exists in the resource permissions array'
+        )
+        it.todo(
+          'should return "false" if "Renameable" property does not exist in the resource permissions array'
+        )
+      })
+
+      describe('canShare', () => {
+        it.todo(
+          'should return "true" if "Shareable" property exists in the resource permissions array'
+        )
+        it.todo(
+          'should return "false" if "Shareable" property does not exist in the resource permissions array'
+        )
+      })
+
+      describe('canCreate', () => {
+        it.todo(
+          'should return "true" if "FolderCreateable" property exists in the resource permissions array'
+        )
+        it.todo(
+          'should return "false" if "FolderCreateable" property does not exist in the resource permissions array'
+        )
+      })
+
+      describe('isMounted', () => {
+        it.todo(
+          'should return "true" if "Mounted" property exists in the resource permissions array'
+        )
+        it.todo(
+          'should return "false" if "Mounted" property does not exist in the resource permissions array'
+        )
+      })
+
+      describe('isReceivedShare', () => {
+        it.todo(
+          'should return "true" if "Shared" property exists in the resource permissions array'
+        )
+        it.todo(
+          'should return "false" if "Shared" property does not exist in the resource permissions array'
+        )
+      })
+    })
+
+    describe('resource icon', () => {
+      it.todo('should be set as "folder" if given resource is a folder')
+      it.todo('should be set as an extension file icon if given resource is not a folder')
+    })
+
+    describe('resource extension', () => {
+      it.todo('should be set as an empty string if given resource is a folder')
+      it.todo("should be set as given resource's file extension if given resource is not a folder")
+    })
+
+    describe('resource type', () => {
+      it.todo('should be set as "folder" if given resource\'s type is "dir"')
+      it.todo('should be set as given resource\'s type if it has value other than "dir"')
+    })
+
+    describe('canDownload', () => {
+      it.todo('should be set as "true" if given resource is not a folder')
+      it.todo('should be set as "false" if given resource is a folder')
+    })
+  })
+
+  describe('attachIndicators', () => {
+    it.todo(
+      'should set resource "indicator" as indicators returned from "getIndicators" helper method'
+    )
+  })
+
+  describe('aggregateResourceShares', () => {
+    // it.todo('')
```

Comment:
@kiranparajuli589 what's the idea here? :)

Reply:
But please don't invest further resources here until we as the `web` team knows what we want - see above

Reply:
i would say skip it or test just the basics, the resources helpers is something we have to refactor

Reply:
@pascalwengerter @fschade sorry I forget to follow up on this PR, actually, this was a draft, I was still not finished and got involved with other things.

Are the added todos good for further implementations? or should we just close it and test only after the refactorings are done.

If it's not required for now, feel free to close it _(or just ping me ;))_.